lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -245,6 +245,7 @@
                                 },
                                 model: team
                             });
+
```

Comment:
Actually, I'd rather drop both the changes to this file since there's no functional changes related to this PR's purpose.